src/demo/modules/circular@1.0.0/chicken.js

```diff
@@ -11,7 +11,7 @@ var chickenNumber = 1;
 Liferay.Loader.define(
 	'circular@1.0.0/chicken',
 	['exports', 'require', './egg'],
-	function(exports, require, egg) {
+	(exports, require, egg) => {
```

Comment:
Almost all of the other code changes are lint autofixes, like this one is.

Reply:
I guess that I should add that we might want to add a suppression to stop this change being applied in directories like this, right? Because if I recall correctly the bundler is going to generate `function()` and not arrows, so it probably makes sense to match the output we get from the bundler.

Reply:
Yes. We should use function() for the tests as it is what the portal expects too. In the end, both are going to work, but it is better to stick to what will be used in production, probably...

Reply:
Ok, updated to leave these `function`s as they were.